projects/components/src/quick-search/quick-search.component.ts

```diff
@@ -195,10 +219,10 @@ export class QuickSearchComponent implements OnInit {
 
     togglePinned(): void {
         this.searchService.isPinned = !this.searchService.isPinned;
-        // When the pinning of the component changes, the searchInput is re-rendered. Because of this, we need to wait for
+        // When the pinning of the component changes, the view is re-rendered. Because of this, we need to wait for
         // Angular to re-render the searchInput before focusing it.
         setTimeout(() => {
-            this.searchInput.nativeElement.focus();
+            this.pinBtn.nativeElement.focus();
```

Comment:
Why did we change where we set the focus? Wasn't the intent to focus the search input?